content/about/resources.fr.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Ressources
+description: 'Ressources | Node.js'
```

Comment:
Usually we recommend using double quotes instead of single ones 👀

Reply:
in the English file it's ' so I left the '

Reply:
I changed this but the problem stay.
do you know where is the problem ?